.flake8

```diff
@@ -1,14 +1,13 @@
-
 [flake8]
 extend-select = C,B,B9,T,AK1
-extend-ignore = E203,E501,B950,E266
+# B905 should be removed once python>=3.10
+extend-ignore = E203,E501,B950,E266,B905
 max-complexity = 100
-exclude = studies, pybind11, rapidjson, docs-*, src/awkward/_typeparser/generated_parser.py, awkward/_typeparser/generated_parser.py
+exclude = studies, pybind11, rapidjson, docs-*, src/awkward/_typeparser/generated_parser.py
```

Comment:
You're right: it's clearly a copy-paste error. Surely excluding the same file twice can't have a useful meaning!